client/src/component/Pages.jsx

```diff
@@ -0,0 +1,263 @@
+import React from 'react';
+import styles from '../componentsStyles/PagesStyle.jsx';
+
+
+class Pages extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {};
+  }
+
+  render() {
+    const {
+      pageNumbers,
+      handlePageClick,
+      backArrowClicked,
+      forwardArrowClicked,
+      currentPage,
+    } = this.props;
+    const {
+      PageButton,
+      ArrowButtons,
+      NumberInButton,
+      SelectedButton,
+      SelectDiv,
+      PageButtonList,
+      PageButtonUL,
+    } = styles;
+    const pageArray = [];
+    const selectedPageButton = (
+      <SelectedButton>
+        <SelectDiv>
+          {currentPage + 1}
+        </SelectDiv>
+      </SelectedButton>
+    );
+    const backArrow = (
+      <ArrowButtons>
+        <svg
+          viewBox="0 0 18 18"
+          role="img"
+          aria-label="Next"
+          focusable="false"
+          style={{
+            height: '0.75em',
+            width: '0.75em',
+            display: 'block',
+            fill: 'rgb(0, 132, 137)',
+          }}>
+          <path d="m4.29 1.71a1 1 0 1 1 1.42-1.41l8 8a1 1 0 0 1 0 1.41l-8 8a1 1 0 1 1 -1.42-1.41l7.29-7.29z" fillRule="evenodd" />
+        </svg>
+      </ArrowButtons>
+    );
+    const forwardArrow = (
+      <ArrowButtons>
+        <svg
+          viewBox="0 0 18 18"
+          role="img"
+          aria-label="Previous"
+          focusable="false"
+          style={{
+            height: '0.75em',
+            width: '0.75em',
+            display: 'block',
+            fill: 'rgb(0, 132, 137)',
+          }}
+        >
+          <path d="m13.7 16.29a1 1 0 1 1 -1.42 1.41l-8-8a1 1 0 0 1 0-1.41l8-8a1 1 0 1 1 1.42 1.41l-7.29 7.29z" fillRule="evenodd" />
+        </svg>
+      </ArrowButtons>
+    );
+    for (let i = 1; i <= pageNumbers; i += 1) {
+      pageArray.push(
+        <PageButtonList>
+          <PageButton type="button" onClick={event => handlePageClick(event, i - 1)}>
+            <NumberInButton>{i}</NumberInButton>
+          </PageButton>
+        </PageButtonList>,
+      );
+    }
+    if (pageArray.length < 3) {
+      return (
+        <PageButtonUL>
+          <PageButtonList>
+            <PageButton type="button" onClick={event => backArrowClicked(event)}>
```

Comment:
backArrowClicked function is not using event...so I would remove it as a parameter.